src/components/task/TasksTable.js

```diff
@@ -100,54 +118,74 @@ export default class TasksTable extends React.Component {
       return item ? item[arrayItemProperty] : '';
     }
 
-    const tableData = this.state.tasks.map(task => [
-      task.id,
-      <Link
-        to={{
-          pathname: `/tasks/${task.id}`,
-          state: {
-            id: task.id,
-          },
-        }}
-        className='text-info'>
-        {task.title}
-      </Link>,
-      getLocalDateFromUTC(task.startDate),
-      getLocalDateFromUTC(task.endDate),
-      `${task.estimation} hours`,
-      <CircleImg
-        logo={
-          task.userId
-            ? findInArray(this.state.users, 'id', task.userId, 'image')
-            : null
-        }
-      />,
-    ]);
+    var orderedTasks = this.state.tasks,
+      tasksByProjects = orderedTasks.reduce((accumulator, taskObject) => {
+        accumulator[taskObject.projectId] =
+          accumulator[taskObject.projectId] || [];
+        accumulator[taskObject.projectId].push(taskObject);
+        return accumulator;
+      }, Object.create(null));
```

Comment:
im a bit confused, why is this stuff in this pull request?

Reply:
Oh yeah! Let me merge develop and it will fix it because I suck at git merging